packages/util/src/find-unserializables.ts

```diff
@@ -0,0 +1,63 @@
+import { notEmpty } from './not-empty';
+
+export interface UnserializableDescription {
+  path: string[];
+  reason: string;
+}
+
+export function findUnserializables(thing: unknown): UnserializableDescription[] | undefined {
+  switch (typeof thing) {
+    case 'number':
+      return isNaN(thing) || thing === Infinity || thing === -Infinity
```

Comment:
nice catch! Instead of checking +- Infinity you can do `!isFinite(thing)`

Reply:
Wow, great catch. Didn't even know about `isFinite`. It also catches `NaN` apparently, so replaced all 3 checks with 1 `isFinite` check 👍  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite#return_value

Reply:
Wow... i knew about isFinite but didn't know it catches NaN as well :D
we all learn hahaha